option optimize_for = LITE_RUNTIME;

message TowerLadderAttrRequest
{
 required uint32 id = 1;
}
